modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -1,12 +1,40 @@
 = Guide to Named Examples in RAML 1.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-If you want to use one or more named examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these three rules:
+If you want to use examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these rules:
 
-* To use a single, named example, you must use the `examples` facet when you include the corresponding NamedExample fragment.
+* Use the `example` facet when you include the `.yaml` file for a single example.
```

Comment:
`For a single example, you should use the `example` facet, and include a `.yaml` file`